src/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import Calculator from './components/Calculator';
 import './index.css';
 
 ReactDOM.render(
-  <App />,
+  <Calculator />,
```

Comment:
- [ ] Great job on the overall implementation. But one thing you must do is to keep the `App.js` file in the project and then render calculator inside the App.js. In all react applications. the App.js is the default entry point for the application and it is where the routes are defined (You'll learn about routing soon). So it is imperative that you keep the App.js file to be able to do the future milestones aswell. Addiitonally, please make sure the App.js is a class based component to conform to the project requirement.

Reply:
@zhadier So what of the index.js should I also keep it